CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-authn-k8s-client#206](https://github.com/cyberark/conjur-authn-k8s-client/issues/206)
 - Improve the error message raised when the username doesn't include the `host/` prefix
   [cyberark/conjur-authn-k8s-client#212](https://github.com/cyberark/conjur-authn-k8s-client/pull/212)
+- The git commit hash is no longer included in the version number of a fixed version.
```

Comment:
Can this be more specific about the impact to the end user of this change? The Docker / RH images before only had the tag version - is what's changing the `version` label in the RH image, or something else? If this doesn't impact the end users, we don't need an entry here.

Reply:
as mentioned above, the affect on the end user is only in the log message - 
Before this PR it represent the commit's hash all the times - 
 CAKC048 Kubernetes Authenticator Client **v0.18.1-e58708b** starting up...

This PR changes it, in case of a fixed version it will be - 
 CAKC048 Kubernetes Authenticator Client **v0.18.1** starting up...

And in case of a non-fixed version it will be as before 
 CAKC048 Kubernetes Authenticator Client **v0.18.1-e58708b** starting up...

so i think that description is correct, agree?

Reply:
it's not clear to me - I might revise it to something like
```suggestion
- The `CAKC048` log message now shows the release version for release builds
   and no longer includes the git commit hash in the log output.
```

Reply:
fixed